static/js/import_wizard/utils/detection.test.tsx

```diff
@@ -0,0 +1,321 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import _, { split } from "lodash";
+
+import { Mapping, TypeProperty } from "../types";
+import * as detection from "./detection";
+
+/* eslint-disable */
```

Comment:
Can we localize these disables to the server responses? The rest of the file should still be linted if possible